macaroonbakery/keys.py

```diff
@@ -5,24 +5,38 @@
 
 
 class PrivateKey(object):
+    ''' A 256-bit Ed25519 public key.
```

Comment:
A private key used by the bakery to encrypt and decrypt third party caveats.
Internally, it is a 256-bit Ed25519 private key.

?